go/oasis-test-runner/oasis/oasis.go

```diff
@@ -249,6 +249,15 @@ type NetworkCfg struct { // nolint: maligned
 	// genesis document for authenticating runtime IDs.
 	IASUseRegistry bool `json:"ias_use_registry,omitempty"`
 
+	// IASMock specifies if Mock IAS Proxy should be used.
```

Comment:
Lots of IAS-related options now, could we group them under a single `IAS` field and have a dedicated structure?